tests/test_reader.py

```diff
@@ -0,0 +1,17 @@
+from scivision.io import load_dataset
+import intake
+
+
+def test_load_dataset_remote():
+    """Test that an intake catalog is generated from scivision.yml file in an example GitHub repo."""
+    assert type(load_dataset('https://github.com/alan-turing-institute/intake-plankton')) == intake.catalog.local.YAMLFileCatalog
+
+
+def test_load_dataset_branch_and_diff_file_name():
+    """Test that an intake catalog is generated when specifying a branch AND that a custom file name."""
+    assert type(load_dataset('https://github.com/alan-turing-institute/intake-plankton/thESciViSionYAMLfileee.yaml', branch='diff-name-yml')) == intake.catalog.local.YAMLFileCatalog
+
+
+def test_load_dataset_local():
+    """Test that an intake catalog is generated from a local yml works."""
+    assert type(load_dataset('tests/test_scivision.yml')) == intake.catalog.local.YAMLFileCatalog
```

Comment:
This works for checking that the catalog loads correctly, but the catalog itself refers to a dataset at a local path that might not exist. Perhaps we could eventually use a dataset that we include? We could then add a series of checks that run the model on it. I think that's for a separate PR though.